lib/src/config/config_file/client.example.toml

```diff
@@ -1,61 +1,73 @@
 ##############################################################################
 #
-#   Nimiq core-rs example configuration file
+#   Nimiq core-rs-albatross example configuration file
 #
 #   Copy this to client.toml and edit appropriately.
 #
-#   The client will look for this config file in:
-#    * './' (current directory)
-#    * '$HOME/.config/nimiq'
+#   The client will look for this config file in '$HOME/.nimiq'
+#   if not specified with the --config option.
 #
 ##############################################################################
 
 ##############################################################################
-#
-# Network specific configuration
-#
+# Network configuration
 ##############################################################################
 
 [network]
 
+# The local multiaddresses this node should bind to. Usually these are loopback addresses.
+# Multiple addresses can be specified.
+#
+# If `advertised_addresses` is not used, these addresses will also be advertised to peers.
 listen_addresses = [
   "/ip4/0.0.0.0/tcp/8443/ws",
+  "/ip6/::/tcp/8443/ws",
 ]
 
+# Seed nodes to connect to when the node starts up. Multiple addresses can be specified.
 seed_nodes = [
   { address = "/dns4/seed1.pos.nimiq-testnet.com/tcp/8443/wss" },
 ]
 
-# Optionally specify address(es) that will be advertised to peers instead of the ones in `listen_addresses`
+# Address(es) that will be advertised to peers instead of the ones in `listen_addresses`.
 #
 # This can be used to advertise the public URL and port that peers should connect to, while
 # `listen_addresses` contains the loopback IP and port that this nodes listens on, which may
 # not be publicly reachable.
-# For validators it is strongly recommended to list a public reachable IPv4 IP.
+# For validators it is strongly recommended to list a public reachable IPv4 IP or a domain that resolves to one.
```

Comment:
```suggestion
Mmm, if you add a domain, it is strongly recommended to add a public reachable IP for autonat to work.
```

Reply:
I cannot find that specific recommendation in the documentation. Can you point to it?

Reply:
It's an autonat limitation: https://github.com/libp2p/specs/blob/master/autonat/README.md#autonat-protocol

Reply:
I read `based on the IP address` that DNS is also allowed.

The more important limitation (that we do not document here at all yet) is that the IP must be the one of the server itself, not another proxy.

Reply:
Complete text is: `implementations MUST NOT dial any multiaddress unless it is based on the IP address the requesting node is observed as`, so there must be an multiaddress based IP since autonat doesn't resolve DNS addresses.

Reply:
I understand 'based on the IP address' as anything that resolves to an IP address. If you say autonat doesn't resolve DNS (which I did not read anywhere), then yes, it must be an IP-multiaddress, ok.

Reply:
In v1, the rust implementation checks by default if the provided address is a global IP: https://github.com/libp2p/rust-libp2p/blob/64c6eb299ce905b2f81fd1317ec616f56dd37986/protocols/autonat/src/v1/behaviour.rs#L331 and since it doesn't resolve DNS, it won't work.

Reply:
Perfekt :+1:

Reply:
Reworded like this: https://github.com/nimiq/core-rs-albatross/compare/8f125169001439461d6f4693af6cc2daf30f4a95..0008b8240d5929558d900633a016911b5305c6af